Notebooks/DC2Cookbook.ipynb

```diff
@@ -393,19 +393,35 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Make the `TemplateCoadd`s\n",
+    "### Make the `SkyMap` [issue [#121](https://github.com/LSSTDESC/DC2_Repo/issues/121)]\n",
+    "\n",
+    "The first thing we need to make is the sky map on which the coadded images will be projected for all bands. In order to ensure that the sky map will not depend on the list of input visits (their location on the sky or on the time and place the reprocessing happens), the sky map should be absolute (using `makeSkyMap.py` and the appropriate configuration) instead of relative to an input field (as done with `makeDiscreteSkyMap.py`). Making such a sky map will create thousands of tracts and patches, that will be fixed on the sky and usable in different reprocessing sessions. \n",
+    "\n",
+    "One possibility could be to use the [Rings sky map](https://github.com/lsst/skymap/blob/master/python/lsst/skymap/ringsSkyMap.py#L42), as done for the HSC data reprocessing with the LSST stack (and currently for CFHT data reprocessing as well). In order to make this sky map, some modifications of the `lsstSimMapper.py` [script](https://github.com/lsst/obs_lsstSim/blob/master/python/lsst/obs/lsstSim/lsstSimMapper.py) must be done to match what has been done in `hasMapper.py` ([here](https://github.com/lsst/skymap/blob/master/python/lsst/skymap/ringsSkyMap.py#L42) and [here](https://github.com/lsst/obs_subaru/blob/master/python/lsst/obs/hsc/hscMapper.py#L286)). A configuration file (`makeSkyMapConfig.py` below) is then needed to run `makeSkyMap.py`, containing information on how to parametrize the rings map. An example of such a configuration can be taken from the HSC reprocessing [repository](https://github.com/LSSTDESC/ReprocessingTaskForce/blob/master/config/w_2017_49/cfht/makeSkyMapConfig.py), which contains:\n",
```

Comment:
We will need to make this a concrete suggestion rather than a statement of what is possible.  Can we make such a suggestion, with associate config, now, or do we need to have more info?

Reply:
I was waiting for reaction from different people and left the choice of the sky map opened on purpose. But I think that using a ring sky map is reasonable, with a configuration close to what was use for HSC.

Reply:
Agreed.  I'm fine closing this with this wording, but we will need to finalize it at some point.